### PR TITLE
restore siggen test

### PR DESCRIPTION
**Pull request by sebastus**
### What does this PR do?
Add a test for the siggen page. Test the operation of the page.
Along the way, it was necessary to change the alt text of the 4 images because they were all the same and that makes it difficult for playwright to find the right one. And, I added the hidden flag to the images because playwright can find it when it becomes visible easily.
Also removed rfdx references.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?

### CODE REVIEW

**Comment by Nepomuceno** [comment]
I think there are more things in this PR them the actual restore of that test.

File: client/src/pages/signalGenerator/SignalGenerator.tsx [hunk 1]
```diff
@@ -417,7 +419,7 @@ print('NumPy Version:', numpy.version.version)
                   theme={vscodeDark}
                 />
                 <br></br>
-                <img src={state.b64ImageIQ} width="490px" alt="hit run to load" />
+                <img src={state.b64ImageIQ} width="490px" alt="IQ tab image" hidden={state.imagesHidden} />
```
Comment (hannahlspencer) [inline client/src/pages/signalGenerator/SignalGenerator.tsx:422]: Love an accurate alt text

**Comment by hannahlspencer** [comment]
Only other comment is agreement with Gab - maybe add more info as to what else this PR does

File: client/src/pages/signalGenerator/SignalGenerator.tsx (hunk 1 above)
Comment (sebastus) [reply client/src/pages/signalGenerator/SignalGenerator.tsx:422]: ty

**Review by hannahlspencer — APPROVED** [review]

**Comment by sebastus** [comment]
I added to the description the other things I changed. These changes were needed to make the test work correctly.

**Comment by Nepomuceno** [comment]
> I added to the description the other things I changed. These changes were needed to make the test work correctly.

I was talking about the docker files that were part of the other pull request but that were merged in this one

**Comment by Nepomuceno** [comment]
> I added to the description the other things I changed. These changes were needed to make the test work correctly.

I was talking about the docker files that were part of the other pull request but that were merged in this one

**Pull request merged**